.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "jest": true
+    },
+    "extends": ["airbnb"],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "no-underscore-dangle": "off",
+      "no-alert": "off",
+      "no-restricted-globals": "off",
+      "import/no-cycle": "off",
+      "no-use-before-define": "off"
+    }
+}
```

Comment:
- This file should look like this: 
![image](https://user-images.githubusercontent.com/46115725/72635164-cc267d80-3921-11ea-9743-1adca4ada260.png)
- Please follow the steps described in the [javascript linter repo](https://github.com/microverseinc/linters-config/tree/master/javascript), to configure the linting for your project, make sure you are using the required configuration, and once that is up and running do the necessary linting (if required).